src/main/scala/ai/nikin/typedgraph/core/Path.scala

```diff
@@ -0,0 +1,12 @@
+package ai.nikin.typedgraph.core
+
+class Path(edges: Set[Edge]) {
+  lazy val from: Vertex = edges.head.from
```

Comment:
Can we rely on the edges Set to be an ordered one? I think we get an unordered Set in the `Path.apply()` method

Reply:
fixed